internal/cli/deployments/search/indexes/create.go

```diff
@@ -105,7 +105,13 @@ func (opts *CreateOpts) RunAtlas() error {
 		return err
 	}
 
-	opts.index, err = opts.store.CreateSearchIndexes(opts.ConfigProjectID(), opts.DeploymentName, index)
+	// TODO: Update to use the new struct
+	result, err := opts.store.CreateSearchIndexes(opts.ConfigProjectID(), opts.DeploymentName, index)
```

Comment:
cc  @fmenezes and @jeroenvervaeke  - for the local run, I'm keeping the old structure of indexes

Reply:
can you create a CLOUDP in the TODO please

Reply:
done https://jira.mongodb.org/browse/CLOUDP-260963

Reply:
can you add a reference here :)

Reply:
done